### PR TITLE
DOC: Add notes for metal GPU acceleration

### PR DESCRIPTION
**Pull request by aresnow1**

### CODE REVIEW

**Review by qinxuye — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -178,6 +178,7 @@ $ xinference list --all
 - Xinference will download models automatically for you, and by default the models will be saved under `${USER}/.xinference/cache`.
 - Foundation models only provide interface `generate`.
 - RLHF and SFT models provide both `generate` and `chat`.
+- If you want to use MacOS Metal GPU for inference, please choose the q4_0 and q4_1 quantization methods.
```
Comment (UranusSeven) [inline README.md:181]:
```suggestion
- If you want to use Apple Metal GPU for acceleration, please choose the q4_0 and q4_1 quantization methods.
```

**Comment by qinxuye** [comment]
Seems modification on readme_zh_CN is absent.

**Comment by aresnow1** [comment]
> Seems modification on readme_zh_CN is absent.

Done.

**Review by qinxuye — APPROVED** [review]
LGTM

**Pull request merged**